docs/source/conf.py

```diff
@@ -69,9 +69,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = "0.14"
+version = "0.13"
 # The full version, including alpha/beta/rc tags.
-release = "0.14.0"
+release = "0.13.2"
```

Comment:
Same as above, I'm just wondering if this version can be found from the package automatically
```suggestion
release = openmc.__version__
```